Lib/fontTools/cffLib/specializer.py

```diff
@@ -220,14 +223,15 @@ def generalizeCommands(commands, ignoreErrors=True):
 			if ignoreErrors:
 				# Store op as data, such that consumers of commands do not have to
 				# deal with incorrect number of arguments.
-				result.append(('', args))
-				result.append(('', [op]))
+				if len(args):
```

Comment:
I don't like this either.  The way I had written the code, it could just passthrough erratic bytecode without changing it.  That was intentional.